new/analyse/create-package-json-content/create-package-json-content.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+  Module,
+  PackageAfterCreatePackageJsonContent,
+  PackageAfterDetermineVersion,
+} from '../../types';
+import { clone } from 'ramda';
+
+import { logger } from '../../helpers/logger';
+import { getReleasablePackages } from '../adapt-dependencies/get-releasable-packages';
+
+import { getPackageJson } from './get-package-json';
+
+const { error, log } = logger('[analyse] create package json content');
+
+export const createPackageJsonContent: Module = (env) => {
+  if (!env.packages) {
+    const errorMessage = '"packages" not present on env config object.';
+    error(errorMessage);
+    throw new Error(errorMessage);
+  }
+
+  log('analyse packageJson Content');
+
+  const releasablePackages = getReleasablePackages(env.packages);
+
+  const clonePackages = clone(env.packages);
+
+  releasablePackages.forEach((packageName) => {
+    log(`generating git & nexus package.json files for ${packageName}`);
+
+    const currentPackage: PackageAfterCreatePackageJsonContent = {
+      ...(clonePackages[packageName] as PackageAfterDetermineVersion),
+      packageJsonGit: getPackageJson(clonePackages, packageName, 'git'),
+      packageJsonNexus: getPackageJson(clonePackages, packageName, 'nexus'),
```

Comment:
Can we please call it NPM (throughout all changes)? As it's open source, not everyone has the same registry type, but all are following the NPM protocol.

Reply:
will do